pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```diff
@@ -41,7 +54,7 @@ stdenv.mkDerivation {
       for scanning directories.
     '';
     homepage = "https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-vulnerability-cli-scanner/";
-    mainProgram = "sysdig-cli-scanner";
+    mainProgram = pname;
```

Comment:
The point of mainProgram was to declare this explicitly and not to copy pname

Reply:
Got it, thank you Sandro